tests/UnitTests/CoreGraphics.drawing/DrawingTest.h

```diff
@@ -16,35 +16,160 @@
 
 #pragma once
 
+#include "ImageComparison.h"
+#include "DrawingTest.h"
+#include "DrawingTestConfig.h"
+#include "ImageHelpers.h"
+
+#include <Starboard/SmartTypes.h>
+#include <memory>
 #include <TestFramework.h>
 #include <CoreGraphics/CoreGraphics.h>
 
 #include <Starboard/SmartTypes.h>
+#include <type_traits>
+
+static const CGSize g_defaultCanvasSize{ 512.f, 256.f };
+
 
 namespace testing {
+template <typename TComparator>
```

Comment:
>TComparator [](start = 19, length = 11)

TComparator=PixelByPixelImageComparator<ComparisonMode::Exact>to simplify things?